deploy/kubernetes/helm-charts/altinn-repositories/templates/service.yaml

```diff
@@ -6,10 +6,8 @@ metadata:
     run: altinn-repositories
 spec:
   ports:
-  {{- range $port := .Values.service.ports}}
-    - port: {{ $port.port }}
-      targetPort: {{ $port.targetPort }}
-      protocol: {{ $port.protocol }}
-  {{- end }}
+    - port: {{ .Values.image.containerPort }}
+      targetPort: {{ .Values.image.containerPort }}
+      protocol: TCP
```

Comment:
This limits us to only export or run one port per deployment/pod. Even though it works for now, maybe we need to export some other ports in the future?

Reply:
Keeping it simple for now... moving to Traefik 2 later.